pydantic/schema.py

```diff
@@ -76,6 +77,7 @@ def schema(
     title: Optional[str] = None,
     description: Optional[str] = None,
     ref_prefix: Optional[str] = None,
+    ref_template: Optional[str] = None,
```

Comment:
@samuelcolvin I was just following the pattern here.
But should I set the `default_template` as the default argument for `ref_template`?

```python
 ref_template: Optional[str] = '#/definitions/{model}',
```

![image](https://user-images.githubusercontent.com/13108583/86518943-9576ed80-be03-11ea-9ef7-62dde041410d.png)

Reply:
I think better to switch to `ref_template: str = '#/definitions/{model}',`, the others are only optional because they really can be optional.

The only advantage of making this optional would be that `ref_template` could take priority over `ref_prefix` if it's not `None`.

Reply:
Done. 
I don't know if it would have been better to keep a `REF_TEMPLATE` constant and use that as the default for all the `ref_template` constant, but it looked like doing that prevented the editor help text from showing what the actual value was.